kovri.i2p/_i18n/fr/contributing.md

```diff
@@ -78,76 +79,96 @@ C4 is meant to provide a reusable optimal collaboration model for open source so
 - Maintainers SHALL have commit access to the repository.
 - Everyone, without distinction or discrimination, SHALL have an equal right to become a Contributor under the terms of this contract.
 
-### Licensing and Ownership
+- Le projet UTILISE le système de révision de contrôle fourni par GIT.
```

Comment:
For this change and next ones, I hesitated but this "should" suggests  that it is used, that's why here and all the following examples, I translated in that way and not in your, I trust you if you think it's better though.